### DISCUSSION TITLE
@wordpress create-block doesn't work out of the box for me

### DISCUSSION THREAD

**Discussion by ZoomREI** [WordPress/gutenberg#62519] [Q&A] [answered]
Hello dear community,

I have a WordPress dev environment that I set up using Docker. Basically I expose 'wp-content' in my directory and the rest of the wp files are stored in the docker volumes.

Anyway, I have a fresh WP installation. 
1. I run `npx @wordpress/create-block@latest my-block --variant dynamic --namespace my-block`. 
2. I go and activate the plugin through the wp-admin dashboard.
3. I open the page editor and my block doesn't appear.

I tried running `npx` with a different plugin name, droping the added command options, deleting browser cache, went over `block.json`, `index.js`, `render.php`, `npm install`, `npm run build` and pretty much anything I could come up with including involving ChatGPT... nothing helped.

I'm pretty hopeless now. Can I get assistance in resolving?

**Comment by talldan** [comment, marked as answer]
@ZoomREI It may be the same issue as discussed here - https://github.com/WordPress/gutenberg/issues/62202

**Reply by ZoomREI** [reply]
Amazing! Thank you. There seems to be some issues with @wordpress/scripts version 28.0.0

This worked:
```
- edit package.json set devedepencies "@wordpress/scripts" to "^27.9.0"
- npm install
- npm start
- refresh and edit a page
- add the new block "/test-plugin"
```
